docs/modules/commons/partials/data-generating.adoc

```diff
@@ -0,0 +1,1642 @@
+[%autowidth.stretch, cols="~a,~a"]
+|===
+|Expression `#{generate(<expression>)}`|Result
+
+|Address.state
+|Louisiana
+
+|Address.country
+|Antigua and Barbuda
+
+|Address.streetAddressNumber
+|930
+
+|Address.streetAddress
+|1197 Roob Lane
+
+|Address.streetAddress 'true'
```

Comment:
I'm ok to merge the current version as the first iteration, however we'll need to describe input parameters, because nobody knows what is `true` here

Reply:
I think it would be more rational to describe the schema and provide users with the sources of data.
Something like this:
![Screenshot 2022-07-19 at 15 15 28](https://user-images.githubusercontent.com/66934999/179741461-671530fb-83c6-4939-b5de-3c9f8e8ed4f0.png)
I can also add these lines if it is OK

Reply:
Looks good, let's add